WinUIGallery/ControlPages/XamlUICommandPage.xaml

```diff
@@ -28,8 +28,7 @@
                               XamlSource="XamlUICommand\XamlUICommandSample1_xaml.txt"
                               CSharpSource="XamlUICommand\XamlUICommandSample1_cs.txt"
                               HorizontalContentAlignment="Stretch"
-                              MinimumUniversalAPIContract="7"
-                              Loaded="ControlExample_Loaded">
+                              MinimumUniversalAPIContract="7">
```

Comment:
What is this change for? Also we should be able to remove any UniversalAPI checks for WinUI3

Reply:
Just some dead code cleanup. If it's better I can also revert this. My thought was though I'm not sure when someone might stumble across this dead code so remove it now before it stays for the next months or even years.